Userland/Libraries/LibFileSystemAccessClient/Client.cpp

```diff
@@ -63,6 +63,9 @@ DeprecatedResult Client::try_request_file(GUI::Window* parent_window, Deprecated
         GUI::ConnectionToWindowServer::the().remove_window_stealing_for_client(child_window_server_client_id, parent_window_id);
     });
 
+    // The endpoint only cares about ReadOnly, WriteOnly and ReadWrite and both enum shares the same layout for these.
+    Core::Stream::OpenMode mode = static_cast<Core::Stream::OpenMode>(deprecated_mode);
```

Comment:
This gives me the heebie-jeebies. I know it's a temporary thing, but could we instead have a static helper function which does this conversion with a `switch`?